configure.ac

```diff
@@ -3551,3 +3567,8 @@ AC_SUBST(GCC_VERSION)
 #--------------------------------------------------------------------
 AC_CONFIG_FILES([config.mak base.make Headers/GNUstepBase/GSConfig.h])
 AC_OUTPUT
+
+# remove copied objc.dll from source directory on Windows
```

Comment:
This seems to be a leftover. Could you please remove it?

Reply:
Good catch, thanks.